File: DTOs/Enums/SalonType.swift
import Foundation

/// Перечисление `SalonType` определяет типы салонов, поддерживаемые в системе.
public enum SalonType: String, Codable, CaseIterable, Equatable {
    case individual  // Означает, что салон является индивидуальным предприятием.
    case chain       // Означает, что салон является частью сети.
    case master      // Означает, что салон управляется мастером.
}

File: android-legacy/shared/src/main/kotlin/shared/dto/enums/SalonType.kt
package shared.dto.enums

import kotlinx.serialization.SerialName

/// Описывает тип Салона
/// - `INDIVIDUAL`: Индивидуальное предприятие
/// - `CHAIN`     : Сеть салонов
/// - `MASTER`    : Самозанятый мастер
enum class SalonType {
    @SerialName("individual")
    INDIVIDUAL,

    @SerialName("chain")
    CHAIN,

    @SerialName("master")
    MASTER
}
